bin/stacks/routing-api-stack.ts

```diff
@@ -206,7 +208,7 @@ export class RoutingAPIStack extends cdk.Stack {
         // For this metric 'avg' represents error rate.
         statistic: 'avg',
       }),
-      threshold: 0.25,
+      threshold: 0.05,
```

Comment:
wow this is pretty aggresive - wonder how you come up with this number?

Reply:
https://github.com/Uniswap/routing-api/pull/93#issuecomment-1267498466